tests/.dirac-ci-config.yaml

```diff
@@ -1,5 +1,8 @@
zlUP4/FwDO2NwAb+jamPCfPNW3HIr4XllML/WAZQJD9TGQLRDxelntictz6n0w57o1shk6CujLNzlcjVQ08hq8ZYpEoM5UTXBC7ZJsIbS6JGODYJQFO2r3haud4e316uUaN/0swDCWqLyLJIEx7BvcP2t002/gtl1WCqaTI1abIKteoTvGINOYF9N6qEmrCCE8hQbIUkxow3f4cKC0XpO5NbloQT/6C7h62XduSiJ0N9SSnaADTLmgTvXO1pP97U5GcNh64E++Fso3ihi9lu9vZwWuuBUMXg4GkPo7A73qspp+9RZrALt6/NRSo028EQrx0gGr4CYySY8m9WFSeb7ocHzud1pc/Ad0oZB6VOo1WTW65mZ52qxUT5fPbIi3O9P2IN3zK35OUwloD0gDezDsMeHsf+d4L6EBVqP4pAMLRRFX6gEjRXEbG5Srd5VX03FZJvL5KzVz54oIkmqTnoIpw6C1Al8rV0V3ypZB3EB+Z3fZzttFxqSWf8zlgzJ0z5Qe5YkJvaQuCNQ6qPj3lzDwhpg2WrcRu4hKGENGQVxVqiKbh9uErNJkRWTeWE80gQkzWmzMq8japL9qyvmM2gpWtX1HLK87d16YA5J0DR5k3NL8LieJ0pbqOdlEdnBvlQPrMjNg8rcsU8Nc9/8IfLSwUKfIHQOzCq3//Uk/6WjaNxE7vAqXeKukb2UQsRpPWAy89j9PdYhQ6jjIYL7yoIHMnnc9vKf6WJNe8KHiQ8Ebz16TK+4W8K0w39kUtu8xj7XZhZhJ6F3COjsybd9frHZI2f7a8QDr1A78jFF1AoFsVNpgP7JXr0JH53jXcuY/Qau0JJMC/bbMFse1+Do6hpfG9Te7D0NNyXL1Mbqh/E999TJXudUV0dUMnd7VrKwM+myYZZovLcoZ2HYmr2FKvAHOLO6vSeuZ41+fCsuqEJ6QMHszUbs1DhAaFcaUgrsnqqatC+wPdplxXFVBkvIUwdIVBOrrmXZPzc4MlvaJmVqP2G4lUmQcvtjFxQNTZHNHF0WhjNHs+RNwEb4uVyd9sVxUOtjX9ebqTf3nB/xRoUp1jWezFw31fekjHI/quyNNrImwuBl99C0/afeMgncnU6GdJif2GDA19Fm79ZAXRdN/+Qt9FEBU8tA9zEWcquSaXxP+OTrtOkVN723ZCc95QqODuFK4Jy39rFlaoin7DtzOZVd1XFzev2wXxmq0OeKFD/+8aRDdbCfQ+YxZWckqiqTiqsK5heJ3OD57PRWLQE7cQdS1PQ2wVctrVekVJNWWrmmhOPuZGe0wyCp8Hj0rkCcsvp7qucU0WFpdD8RDcjkUAD7QigWCn0sowSAZlcLATKEMxMYT60BLKBlh0onarwKn5mF2Z9EoFhmfMl730T4FGc8Hd8nXdCPviAhIa2HOwyMXv5+5RBEWAW1k42Z28Whrw/E25dExkaPtMDCuTnBhCZpLhmpMk5yiL16b5ZF60Iua8FEubExdoyQRVGbFnCw+PzmqnJarYX7+pgtjjE9l9nsCqCuZ6G28My9tXEXSV6roe68vkf7TcEdSKQTZTyeSXbIIikTMJ8cA0BLfPd1eOKj9cFBM1Byn411LiUMsd3maPuj4zoviMVvG2AO3l84N/gL+qNKoeU3Lcx/GQIUYpFWYQwwzbpy5FgKAEL4/p67dl0k7ROubCOgHTDgvEYCgyCm+QcCaqyYmGFJsKX+GuUpLLdO2Gniq23HFnqc097PoMuUMYAqhyTR4RWvrui25+Qs0Th5G+9iTgdEhr2kk36mLCgJWgiDqRMv1e7pdA+a5Hb9S7bvkt3xeg5HVQf2x/rNuhLTEhOMIFbbNOOqpO/WXYGyzFh/oVg7zb+DYKooYUCLV2TFdLaRCJQNpoZXlzq6O9ughPXZO4ruSEKrzv5MfoCU83OPtbmvvaBx6r2bNSxMo3FHH3yoftoJupCSV6sBZNZfWDNtz23xdcJ4vg4tNe2xv2pXjRsA1WqvVPciL3prDp46o4KPKF2UlSf8Q3J5oEmx+3/8HRV5zEhXYv4aX7ncVIPu0Y127HeAcDnmUMY7VLS7zEH35DC169mJC2pP+dGl6PY3aPF4yBXitHl+bsmoZo2Opatm6Z1fLyEW4d9oxYjCfD60TuhztSRK7XXz0SQv0csiiECR4z0xDorWeR5QOQIF69Gutyk+SK27KtHv7Y9zsN3S8KN/XPXZsgU6eJT+2cDsVMcgzCcNK219RIctxZw3i/rexz3XoiIC1PiVt3g2TzCAZz8KA3JnSlMp/+sJBhm+n0Y2sSalqUnBAVOhiFVDvBpGGCJdQwC3aQmSnwLhgrTfUNNGeGCuRvFBFPqvrTWCnGHKekWT179mNI6rmfsY/VP49ybqBe/8YbKqshZkzmTYR/SxBuOosHV3BQ314faayZBqgakEBDcQL4QZVixxJM1nVz4UBXLn1dBv3PHNHA1tdiWRoTZW8gbpVnhE3u2L3ET4bdf1MHZ+nF991laGmOua7ltoUdFi2KVq4Kf+8dWeBtsX4txnrUPL9K+g9ras/GAMX9OWB+5gkjY1zkX57JUGIVDToIjuN/9nveXkP0HClgb3KWNXY8QRtprECsuY3PlJ4FaxUIAAABqt0neAAHDJeiKAQBz3nq7PjANiwIAAAAAAVlaAAAAvgAEAAAAfXXY/nQAQQABAAAc/gAAAGQFVEZpbGUQcmVkdWNlZF9maWxlLmRzdA9Sb290IGV2ZW50IGRhdGEAAAACAAACSgAEAAAJIXXY/nQAOQABAAAE6gAAAGQFVFRyZWUEUmVmcxNSb290IHJlZmVyZW5jZSBkYXRhAAACmQAEAAAK6HXY/nQAQAABAAAHNAAAAGQFVFRyZWULRmlsZVJlY29yZHMTUm9vdCBkYXRhIGZvciBHYXVkaQAAAEsABAAAAAp12P50AEEAAQAAHbwAAABkBVRGaWxlEHJlZHVjZWRfZmlsZS5kc3QPUm9vdCBldmVudCBkYXRhAAEAAB4HdzWUAA==
```

Comment:
Why is this here? At a glance it just seems to be a file for testing, but if so why not generate random data on demand.

Committing random binary blobs makes me think of https://tukaani.org/xz-backdoor/ 😉

Reply:
This is an LHCb DST file :-) with only 3 events in it (I need it for an LHCb test). I can move it to LHCb and here just put something simpler.